api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -11,11 +11,52 @@ import "google/protobuf/empty.proto";
 import "validate/validate.proto";
 
 import "envoy/config/route/v3/route_components.proto";
+import "envoy/type/matcher/v3/string.proto";
 
 message FilterTransformations {
   // Specifies transformations based on the route matches. The first matched transformation will be
   // applied. If there are overlapped match conditions, please put the most specific match first.
   repeated TransformationRule transformations = 1;
+
+  // Only RouteTransformations.Transformations with matching stage will be used with this filter.
+  uint32 stage = 2 [(validate.rules).uint32 = {lte: 10}];
+}
+
+message RouteTransformations {
+  // deprecated
+  Transformation request_transformation = 1;
```

Comment:
```suggestion
  Transformation request_transformation = 1 [deprecated=true];
```

Reply:
Does this add the Go comment as well, the `Deprecated`?

Reply:
@EItanya yeah i believe it adds the deprecated signature for whichever language you're generating code from